.circleci/config.yml

```diff
@@ -42,3 +46,10 @@ jobs:
           command: |
             if [ "<< parameters.xcode >>" = "12.0.0" ]; then export XCODE_XCCONFIG_FILE=~/project/xcode-12-fix.xcconfig; fi
             Tests/Integration/test_carthage.sh
+      - when:
+          condition:
+            equal: [ "11.7.0", << parameters.xcode >> ]
```

Comment:
Is there any problem with XCode 12 code coverage?

Reply:
Not that I know of but since Xcode 12 is still in beta and we run the same tests as with Xcode 11, I figured having multiple reports would be excessive at this point.